hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/handlers/marker/MarkerDirState.java

```diff
@@ -270,9 +272,11 @@ private void syncMarkersFromFileSystem() {
     try {
       if (fileSystem.exists(dirPath)) {
         FileStatus[] fileStatuses = fileSystem.listStatus(dirPath);
+        Predicate<String> prefixFilter = pathStr -> pathStr.contains(MARKERS_FILENAME_PREFIX);
```

Comment:
Is it possible to re-use MarkerUtils.readTimelineServerBasedMarkersFromFileSystem() ?